packages/opencensus-exporter-zpages/src/zpages.ts

```diff
@@ -65,21 +77,56 @@ export class ZpagesExporter implements Exporter {
   }
 
   /**
-   * Is called whenever a span is started.
+   * It's called whenever a span is started.
```

Comment:
Nit: Remove `It's` and just have it say: "Called whenver a span is started."

Reply:
Done